fe/fe-core/src/main/java/org/apache/doris/mysql/privilege/Auth.java

```diff
@@ -317,6 +320,12 @@ public boolean checkTblPriv(UserIdentity currentUser, String ctl, String db, Str
     public void checkColsPriv(UserIdentity currentUser, String ctl, String db, String tbl, Set<String> cols,
             PrivPredicate wanted) throws AuthorizationException {
         // TODO: Support column priv
+        // we check if have tbl priv,until internal support col auth.
+        if (!checkTblPriv(currentUser, ctl, db, tbl, wanted)) {
```

Comment:
Need to update the case:
`account_p0.test_nereids_authentication`

See: http://43.132.222.7:8111/buildConfiguration/Doris_DorisRegression_P0Regression/116156?hideProblemsFromDependencies=false&hideTestsFromDependencies=false&expandPull+Request+Details=true&expandBuildProblemsSection=true&expandBuildTestsSection=true